WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -373,6 +373,16 @@ public void onEventMainThread(PostEvents.PostUploadStarted event) {
         }
     }
 
+    /*
+    * Upload cancelled (probably due to failed media), reload so correct status on uploading post appears
+    */
+    @SuppressWarnings("unused")
+    public void onEventMainThread(PostEvents.PostUploadCanceled event) {
+        if (isAdded() && mSite.getId() == event.localSiteId) {
```

Comment:
Agreed with the null check - note though that this isn't a FluxC callback - it's the old WordPress-internal event bus we still use for some internal actions.